packages/SwingSet/src/liveslots/vatstore-usage.md

```diff
@@ -38,18 +38,17 @@ Vats can use `VatData.defineKind()` to define categories (classes) of virtual da
 
-`defineKind` also specifies the runtime behavior: an `init` function called to create the initial state of each instance, a `behavior` record to provide methods for each instance, and an optional `finish` function to perform post-initialization tasks like registering the new object in a collection. `defineKind` returns a "kind constructor", typically named e.g. `makeFoo()` to make instances of the "foo kind.
+`defineKind` also specifies the runtime behavior: an `init` function called to create the initial state of each instance, a `behavior` record to provide methods for each instance, and an optional `finish` function to perform post-initialization tasks like registering the new object in a collection. `defineKind` returns a "kind constructor", typically named e.g. `makeFoo()` to make instances of the "foo" kind.
 
 Each time the kind constructor is called, a new "baseref" is allocated for the cohort, and the (one or multiple) facet Representatives are created. Each Representative/Facet gets a separate vref, all of which are extensions of the baseref.
+Those vrefs are used when interacting with the kernel (in `syscall.send` etc.), and in virtualized data (inside the capdata `slots` that point to other objects), but the vatstore keys that track GC refcounts and export status use the "baseref" instead.
 
-The vref is used when interacting with the kernel (in `syscall.send` etc), and in virtualized data (inside the capdata `.slots` that point to other objects). The vatstore keys that track GC refcounts and the export status use the "baseref" instead.
+The baseref is built out of two pieces separated by a literal `/`:
+1. Prefixed Kind ID, e.g. `o+11`. These are allocated from the same "Export ID" numberspace as exported Remotables (JavaScript objects marked with `Far`).
+2. Instance ID, an integer. `1` for the first instance of each Kind and incremented for each subsequent instance.
 
-The vrefs are built out of three pieces:
-* Kind ID, e.g. `o+11`. These are allocated from the same "Export ID" numberspace as exported Remotables (JS `Object`s marked with `Far`).
-* Instance ID, an integer, "1" for the first instance of each Kind, incrementing thereafter
-* Facet ID, missing for single-facet Kinds, else an integer starting with "0"
-
-These are combined with simple delimiters: `/` between the Kind ID and the Instance ID, and `:` before the Facet ID (if any).
+The vref for a Representative of a single-facet Kind is just the `o+${kindID}/${instanceID}` baseref. The vref for a Representative of a facet of a multi-facet Kind extends that to `o+${kindID}/${instanceID}:${facetID}`, where the additional component is:
+3. Facet ID, an integer. `0` for the first facet and incremented for each subsequent facet.
```

Comment:
It's interesting that Facet IDs count from 0 while Kind IDs, Instance IDs, etc. count from 1. Should that discrepancy be highlighted, or is it basically irrelevant?